gen/templates/data_source.go

```diff
@@ -152,7 +152,7 @@ func (d *{{camelCase .Name}}DataSource) Schema(ctx context.Context, req datasour
 func (d *{{camelCase .Name}}DataSource) ConfigValidators(ctx context.Context) []datasource.ConfigValidator {
     return []datasource.ConfigValidator{
         datasourcevalidator.ExactlyOneOf(
-            path.MatchRoot("id"),
+            path.MatchRoot("id"), // is this correct?
```

Comment:
We can remove the comment for now. "id" is correct, though we would need to test import separately.

Reply:
removed